travis.sh

```diff
@@ -24,6 +24,7 @@ test_rdmd() {
 
 build_digger() {
     git clone --recursive https://github.com/CyberShadow/Digger "$DIGGER_DIR"
```

Comment:
Currently we don't use any caching on Travis so putting the version/hash in the directory might be a bit unnecessary, though I wouldn't mind as it's a good practice.